File: SofiForce.Models/Models/ListModels/ClientActivityListModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ClientActivityListModel
    {

        public long ActivityId { get; set; }
        public int? ClientId { get; set; }
        public int? ActivityTypeId { get; set; }
        public DateTime? ActivityDate { get; set; }
        public DateTime? ActivityTime { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public bool? IsPositive { get; set; }
        public string ClientCode { get; set; }
        public string ClientName { get; set; }
        public int? RepresentativeId { get; set; }
        public int? Duration { get; set; }
        public bool? InJourney { get; set; }
        public decimal? Distance { get; set; }
        public long? SalesId { get; set; }
        public string RepresentativeCode { get; set; }
        public string RepresentativeName { get; set; }
        public string BranchCode { get; set; }
        public string BranchName { get; set; }
        public int? BranchId { get; set; }

        public bool? InZone { get; set; }
        public DateTime? CallAgain { get; set; }


    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ClientActivityListModel.ts
export class ClientActivityListModel {
    activityId: number;
    clientId: number | null;
    activityTypeId: number | null;
    activityDate: Date | null;
    activityTime: Date | null;
    latitude: number | null;
    longitude: number | null;
    isPositive: boolean | null;
    clientCode: string;
    clientName: string;
    representativeId: number | null;
    duration: number | null;
    inJourney: boolean | null;
    distance: number | null;
    salesId: number | null;
    representativeCode: string;
    representativeName: string;
    branchCode: string;
    branchName: string;
    branchId: number | null;
    inZone: boolean | null;
    callAgain: Date | null;
}